### PR TITLE
Change style of inline code.

### PR DESCRIPTION
**Pull request by gigamonkey**
Let me know what you think of this. Removed the background color and the extra padding and darkened the color a bit.

### CODE REVIEW

File: _static/css/custom.css [hunk 1]
```diff
@@ -55,6 +55,14 @@ section section h2,
   margin-bottom: 0.5em;
 }
 
+.runestone-sphinx code
```
Comment (bhoffman0) [inline _static/css/custom.css:58]:
```suggestion
code, 
.runestone-sphinx code
```

**Review by bhoffman0 — COMMENTED** [review]
Looks good, but not enough contrast in dark mode

**Comment by gigamonkey** [comment]
Latest change at least passes the accessibility checker. Dunno about it's aesthetics but maybe it's okay.

**Review by bhoffman0 — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**